.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - fix_cross_compile
```

Comment:
Before you merge this PR, please remove this line.

Reply:
Yes. Of course.